src/marqo/core/models/marqo_index.py

```diff
@@ -0,0 +1,82 @@
+from enum import Enum
+from typing import List, Optional, Dict, Any
+
+from pydantic import BaseModel
+
+
+class IndexType(Enum):
+    Typed = 'typed'
+    Dynamic = 'dynamic'
+
+
+class FieldType(Enum):
```

Comment:
are these Marqo or Vespa fieldtypes? it would be worth making this explicit (e.g.: `MarqoFieldType`)